javascript/javascript1/week2/addCandy.js

```diff
@@ -0,0 +1,44 @@
+let boughtCandyPrices = [];
+let amountToSpend = Math.random() * 100;
+function addCandy(candyType, weight) {
+  if (candyType === "sweet") {
+    let price1 = 0.5 * weight;
+    boughtCandyPrices.push(price1);
+    return;
+  }
+  if (candyType === "Chocolate") {
+    let price2 = 0.7 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+  if (candyType === "Toffee") {
+    let price2 = 1.1 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+  if (candyType === "Chewing-gum") {
+    let price2 = 0.03 * weight;
+    let n = boughtCandyPrices.push(price2);
+    return;
+  }
+}
+addCandy("sweet", 20);
+addCandy("Chocolate", 100);
+console.log(boughtCandyPrices);
+console.log(amountToSpend);
+
+function canBuyMoreCandy() {
```

Comment:
can you try to write it again using a parameter of a `candyType` so you can call it after like this
```js
canBuyMoreCandy('sweet');
```